sotodlib/preprocess/processes.py

```diff
@@ -141,8 +144,11 @@ def select(self, meta, proc_aman=None):
         else:
             raise ValueError(f"Entry '{self.select_cfgs['kind']}' not"
                                 "understood. Expect 'any' or 'all'")
-        meta.restrict("dets", meta.dets.vals[keep])
-        return meta
+        if in_place:
+            meta.restrict("dets", meta.dets.vals[keep])
+            return meta
+        else:
+            return keep
```

Comment:
since this 5 line block is copied in every select step should we make a method of the Pipeline class called `restrict` or `cut` that takes in `meta`, `in_place`, and `keep` and then make these 5 lines: `return self.cut(meta, keep, in_place)`?